msgid ""
msgstr ""
"Project-Id-Version: ubuntu-translations\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2023-06-09 17:35+0000\n"
"PO-Revision-Date: 2024-11-06 10:17\n"
"Last-Translator: Ubuntu Belarusian Translators Team <Unknown>\n"
"Language-Team: Ukrainian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2024-11-05 06:34+0000\n"
"X-Generator: Launchpad (build 22ade00ab50b929fac63b8ee7252243aceda294a)\n"
"Plural-Forms: nplurals=4; plural=((n%10==1 && n%100!=11) ? 0 : ((n%10 >= 2 && n%10 <=4 && (n%100 < 12 || n%100 > 14)) ? 1 : ((n%10 == 0 || (n%10 >= 5 && n%10 <=9)) || (n%100 >= 11 && n%100 <= 14)) ? 2 : 3));\n"
"X-Crowdin-Project: ubuntu-translations\n"
"X-Crowdin-Project-ID: 485161\n"
"X-Crowdin-Language: uk\n"
"X-Crowdin-File: /main/noble/en_GB/po-usb-creator_usbcreator-be.po\n"
"X-Crowdin-File-ID: 57\n"
"Language: uk_UA\n"

#: .././gui/usbcreator-gtk.ui.h:1
msgid "Installation Failed"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:2
msgid "The installation failed.  Please see ~/.cache/usb-creator.log for more details.\n"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:4
msgid "Installation Complete"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:5
msgid "Test Disk"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:6
msgid "Installation is complete.  You may now run Ubuntu on other computers by booting them with this drive inserted."
msgstr ""

#. #-#-#-#-#  usbcreator.pot (PACKAGE VERSION)  #-#-#-#-#
#. set title of progress window (same as gtk frontend)
#: .././gui/usbcreator-gtk.ui.h:7 .././usbcreator/frontends/kde/frontend.py:446
#: usbcreator/frontends/kde/frontend.py:446
msgid "Installing"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:8 .././usbcreator/frontends/kde/frontend.py:522
#: usbcreator/frontends/kde/frontend.py:522
msgid "Quit the installation?"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:9 .././usbcreator/frontends/kde/frontend.py:523
#: usbcreator/frontends/kde/frontend.py:523
msgid "Do you really want to quit the installation now?"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:10 usbcreator-kde.ui.py:3
#: usbcreator-kde.ui.py:36
msgid "Make Startup Disk"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:11 usbcreator-kde.ui.py:6
msgid "To try or install Ubuntu from a removable disk, it needs to be set up as a startup disk."
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:12 usbcreator-kde.ui.py:9
msgid "Source disc image (.iso):"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:13 usbcreator-kde.ui.py:21
msgid "Other..."
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:14 usbcreator-kde.ui.py:24
msgid "Disk to use:"
msgstr ""

#: .././gui/usbcreator-gtk.ui.h:15
msgid "Erase Disk"
msgstr ""

#: .././desktop/usb-creator-kde.desktop.in.h:1
#: .././desktop/usb-creator-gtk.desktop.in.h:1
msgid "Startup Disk Creator"
msgstr ""

#: .././desktop/usb-creator-kde.desktop.in.h:2
#: .././desktop/usb-creator-gtk.desktop.in.h:2
msgid "Create a startup disk using a CD or disc image"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:1
msgid "Image the device"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:2
msgid "System policy prevents writing a disk image to this device"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:3
msgid "Run KVM"
msgstr ""

#: .././dbus/com.ubuntu.usbcreator.policy.in.h:4
msgid "System policy prevents running KVM"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:217
#: .././usbcreator/frontends/gtk/frontend.py:655
#: usbcreator/frontends/kde/frontend.py:217
msgid "Installation failed."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:372
#: .././usbcreator/frontends/gtk/frontend.py:430
#: usbcreator/frontends/kde/frontend.py:372
msgid "The device is not large enough to hold this image."
msgstr ""

#. #-#-#-#-#  usbcreator.pot (PACKAGE VERSION)  #-#-#-#-#
#. This here filter is for kfiledialog, no clue if it will ever make a return
#. filter = '*.iso|' + _('CD Images') + '\n*.img|' + _('Disk Images')
#: .././usbcreator/frontends/kde/frontend.py:404
#: .././usbcreator/frontends/gtk/frontend.py:548
#: usbcreator/frontends/kde/frontend.py:404
msgid "CD Images"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:404
#: .././usbcreator/frontends/gtk/frontend.py:549
#: usbcreator/frontends/kde/frontend.py:404
msgid "Disk Images"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:434
#: .././usbcreator/frontends/gtk/frontend.py:577
#: usbcreator/frontends/kde/frontend.py:434
msgid "Are you sure you want to write the disc image to the device?"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:435
#: .././usbcreator/frontends/gtk/frontend.py:578
#: usbcreator/frontends/kde/frontend.py:435
msgid "All existing data will be lost."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:444
#: usbcreator/frontends/kde/frontend.py:444
msgid "Cancel"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:455
#: usbcreator/frontends/kde/frontend.py:455
msgid "Starting up"
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:464
#: usbcreator/frontends/kde/frontend.py:464
msgid "You must select both source image and target device first."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:475
#: .././usbcreator/frontends/gtk/frontend.py:611
#: usbcreator/frontends/kde/frontend.py:475
msgid "Finishing..."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:494
#: usbcreator/frontends/kde/frontend.py:494
msgid "The installation is complete.  You may now reboot your computer with this device inserted to try or install Ubuntu."
msgstr ""

#: .././usbcreator/frontends/kde/frontend.py:506
#: usbcreator/frontends/kde/frontend.py:506
msgid "Retry?"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:342 usbcreator-kde.ui.py:12
msgid "CD-Drive/Image"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:355 usbcreator-kde.ui.py:15
msgid "OS Version"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:365 usbcreator-kde.ui.py:18
msgid "Size"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:500 usbcreator-kde.ui.py:27
msgid "Device"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:516 usbcreator-kde.ui.py:30
msgid "Label"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:526 usbcreator-kde.ui.py:33
msgid "Capacity"
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:584
msgid "Starting up..."
msgstr ""

#: .././usbcreator/frontends/gtk/frontend.py:609
#, python-format
msgid "%d%% complete"
msgstr ""

#: .././usbcreator/install.py:92
#, python-format
msgid "The extension \"%s\" is not supported."
msgstr ""

#: .././usbcreator/install.py:101
#, python-format
msgid "An uncaught exception was raised:\n"
"%s"
msgstr ""

#: .././usbcreator/install.py:105
msgid "Writing disk image..."
msgstr ""

#: .././usbcreator/install.py:106
#, python-format
msgid "Could not write the disk image (%(source)s) to the device (%(device)s)."
msgstr ""

#: .././bin/usb-creator-kde:55
msgid "provide a source image to pre-populate the UI."
msgstr ""

#: .././bin/usb-creator-kde:60
msgid "allow writing to system-internal devices"
msgstr ""

